docs/02-app/02-api-reference/05-next-config-js/devIndicators.mdx

```diff
@@ -31,16 +79,30 @@ module.exports = {
 }
 ```
 
-There is also an indicator to show whether a page will be prerendered during a build in dev. It can be hidden per-page by clicking although if you never want it to show it can be disabled:
+## Troubleshooting
 
-```js filename="next.config.js"
-module.exports = {
-  devIndicators: {
-    appIsrStatus: false,
-  },
-}
+### Static route not showing the indicator
+
+If you expect a route to be static and the indicator is enabled but not showing, it's likely the route has opted out of static rendering.
+
+You can confirm if a route is [static](/docs/app/building-your-application/rendering/server-components#static-rendering-default) or [dynamic](/docs/app/building-your-application/rendering/server-components#dynamic-rendering) by building your application using `next build`, and checking the output in your terminal. Static (or prerendered) routes will display a `○` symbol, whereas dynamic routes will display a `ƒ` symbol. For example:
```

Comment:
```suggestion
You can confirm if a route is [static](/docs/app/building-your-application/rendering/server-components#static-rendering-default) or [dynamic](/docs/app/building-your-application/rendering/server-components#dynamic-rendering) by building your application using `next build --debug`, and checking the output in your terminal. Static (or prerendered) routes will display a `○` symbol, whereas dynamic routes will display a `ƒ` symbol. For example:
```
The `--debug` flag will show the actual reason for us bailing out of static.

Reply:
Oh nice!